### PR TITLE
Add flag to avoid recreating the development velum image on every run

### PR DESCRIPTION
**Pull request by ereslibre**

### CODE REVIEW

**Comment by MaximilianMeister** [comment]
it might already work with `SKIP_IMAGE_UPDATE` but I haven't checked that fully

https://github.com/kubic-project/caasp-devenv/blob/master/start#L95

**Comment by ereslibre** [comment]
> wouldn't it be better to add a NO_REBUILD flag to https://github.com/kubic-project/caasp-devenv and pass it through from here?

Take into account that this is not a problem of the `caasp-devenv` but of https://github.com/kubic-project/e2e-tests/blob/master/scripts/cleanup_environment#L14-L17, that upon cleanup, it will remove the tags (the development environment does not untag images).

File: README.md [hunk 1]
```diff
@@ -46,6 +46,10 @@ achieved by specifying the following environment variables:
 - TERRAFORM_BRANCH: branch to use for `k8s-terraform`
 - SALT_BRANCH: branch to use for `k8s-salt`
 - CAASP_CONTAINER_MANIFESTS_BRANCH: branch to use for `caasp-container-manifests`
+- SKIP_VELUM_IMAGE_CLEANUP: by default the velum image is rebuilt on every run for extra
```
Comment (MaximilianMeister) [inline README.md]: I'd maybe choose a different headline for that documentation, as it's not branch related

Comment (ereslibre) [inline README.md]: You're right! my bad!

**Pull request merged**